server/templates/insights.html

```diff
@@ -26,6 +26,7 @@
  {% block head %}
  <link rel="stylesheet" href={{url_for('static', filename='css/insights.min.css', t=config['VERSION'])}} >
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+ <script src="https://datacommons.org/datacommons.js"></script>
```

Comment:
Remove?

Reply:
This actually needs to stay in now for the charts to show up. It looks like the double-defining issue got fixed in one of the PRs, or was just a different mistake on my end.